Dockerfile

```diff
@@ -0,0 +1,18 @@
+# TODO: https://snyk.io/blog/best-practices-containerizing-python-docker/
+
+FROM python:3.10-slim
+
```

Comment:
If we're pushing this to the azure register, could be nice add the label *AUTHOR* or *MAINTAINER*:
```
LABEL AUTHOR="Proyecto Nutria <contact@proyectonutria.com>"
```
If u add something like that, we can add the required labels later on the *hadolint* hook (maybe add something else like *LICENSE* or so).